advocacy_docs/partner_docs/ThalesGuide/05-UsingCTE.mdx

```diff
@@ -1,25 +1,28 @@
 ---
-title: 'Using CipherTrust Transparent Encryption (CTE)'
+title: '4. Using CipherTrust Transparent Encryption (CTE)'
```

Comment:
No need to surround Postgres in parentheses 
No need for the comma before "and"